### PR TITLE
feat(Touch): add a new behavior about touch and go to location

### PR DESCRIPTION
**Pull request by ronal2do**

### CODE REVIEW

File: src/Slider.js [hunk 1]
```diff
@@ -426,9 +450,11 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
 
   _thumbHitTest = (e: Object) => {
     const nativeEvent = e.nativeEvent;
-    this.changeThumSize(24);
     const thumbTouchRect = this._getThumbTouchRect();
-    this._bubble.press();
+    
+    this._setCurrentValue(Math.floor(100 * nativeEvent.locationX / this.state.containerSize.width));
```
Comment (Henawey) [inline src/Slider.js]: what 100 means? can you move it to constant with a meanigful name?

File: src/Slider.js [hunk 2]
```diff
@@ -535,14 +561,16 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
...
           ref={(bubble: any) => this._bubble = bubble}
           value={this.props.value}
           thumbTintColor={thumbTintColor}
-          style={{
+          style={[{
             position: 'absolute',
             transform: [{ translateX: thumbLeft }, { translateY: 0 }],
-          }}
+          }, 
+          this.props.value < 1 ? {display: 'none'} : {},
```
Comment (Henawey) [inline src/Slider.js:575]: what 1 mean? can you move it to constant with a meaningful name?

File: src/Slider.js [hunk 3]
```diff
@@ -257,14 +268,20 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
   }
 
   _handlePanResponderGrant = () => {
+    
+    if (!this.isMoving) {
+      this.isMoving = true;
+    }
     this._previousLeft = this._getThumbLeft(this._getCurrentValue());
+    this.changeThumSize(24);
```
Comment (Henawey) [inline src/Slider.js]: same here for the 24!

**Review by Henawey — APPROVED** [review]

File: src/Slider.js (hunk 1 above)
Comment (ronal2do) [reply src/Slider.js]: 100% of bar width

File: src/Slider.js (hunk 3 above)
Comment (ronal2do) [reply src/Slider.js]: ok

File: src/Slider.js (hunk 2 above)
Comment (ronal2do) [reply src/Slider.js:575]: ok

File: src/Slider.js [hunk 4]
```diff
@@ -237,7 +241,16 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
...
     if (this.props.value !== prevValue) {
-      this._setCurrentValueAnimated(this.props.value);
+      this._setCurrentValueAnimated(this.props.value); 
+      if (this.isMoving) {
+        if (!this.isPressed) {
+          this._bubble.press();
+          this.isPressed = true;
+        }
+      }
+      if(!this.isMoving) {
```
Comment (Henawey) [inline src/Slider.js:251]: I think we can replace this with `else`

**Pull request merged**